packages/common/nbstore/src/sync/blob/index.ts

```diff
@@ -97,7 +98,7 @@ export class BlobSyncImpl implements BlobSync {
     return combineLatest(
       this.peers.map(peer => peer.blobPeerState$(blobId))
     ).pipe(
-      throttleTime(1000),
+      throttleTime(1000, asyncScheduler, { trailing: true }),
```

Comment:
just leave scheduler undefined 

```suggestion
      throttleTime(1000, undefined, { trailing: true }),
```